### PR TITLE
Prod rework

### PR DESCRIPTION
**Pull request by jaspnas**

### CODE REVIEW

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    volumes:
+        - ./nginx.conf:/etc/nginx/conf.d/default.conf
+
+volumes:
+  date_postgres_data:
+
+#networks: 
+#  default: 
+#    external: 
+#      name: external-example
```
Comment (theoabw) [inline docker-compose.prod.yml:103]: Does this need to be here?

File: docker-compose.prod.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      - ALLOWED_HOSTS=${ALLOWED_HOSTS}
+      - ALLOWED_ORIGINS=${ALLOWED_ORIGINS}
+      - EXTRA_STAFF_GROUPS=${EXTRA_STAFF_GROUPS}
+      - EMAIL_HOST_RECEIVER=${EMAIL_HOST_RECEIVER}
+      - DEFAULT_FROM_EMAIL=${DEFAULT_FROM_EMAIL}
+      - EMAIL_HOST=${EMAIL_HOST}
+      - EMAIL_HOST_USER=${EMAIL_HOST_USER}
+      - EMAIL_HOST_PASSWORD=${EMAIL_HOST_PASSWORD}
+      - CF_TURNSTILE_SITE_KEY=${CF_TURNSTILE_SITE_KEY}
+      - CF_TURNSTILE_SECRET_KEY=${CF_TURNSTILE_SECRET_KEY}
```
Comment (theoabw) [inline docker-compose.prod.yml:38]: Consider using env_file directive with separate env files for each container if needed. Could make the compose files cleaner. "web" and "asgi" seems to have the same env vars defined so they could maybe have a common env file that's defined with the env_file directive.

Comment (jaspnas) [reply docker-compose.prod.yml:38]: Topic for a later issue, problem is some variable names are different for host and container and all env variables are currently sourced into host env first. 

Probably should be tackled in #348

File: docker-compose.prod.yml (hunk 1 above)
Comment (andreedd) [reply docker-compose.prod.yml:103]: Nope, I think this was some old network conf was from when I tried some Minio stuff :D

**Review by andreedd — COMMENTED** [review]
Looks pretty clean overall!

And yep would probably be a good idea to clean up env variables to some .env file if still continuing with compose

**Review by Theoabw — APPROVED** [review]
lgtm as long as the clutter get cleaned up at some point. You could add comments about this to #348 and merge.

**Comment by jaspnas** [comment]
> lgtm as long as the clutter get cleaned up at some point. You could add comments about this to #348 and merge.

Created #528 and #529 to address this

File: docker-compose.prod.yml (hunk 2 above)
Comment (andreedd) [reply docker-compose.prod.yml:38]: We could also start using something like doppler for secrets.
https://www.doppler.com/

Comment (andreedd) [reply docker-compose.prod.yml:38]: Free subscription for students :)
All the env variables should also have default values in the app, e.g. `.get(EMAIL_HOST_USER, 'admin@example.com')`

Comment (jaspnas) [reply docker-compose.prod.yml:38]: Tracked in #531 . Will probably be done later to limit scope of project

**Pull request closed without merging**